extensions/wikia/AdEngine/js/lookup/prebid/adapters/indexExchange.js

```diff
@@ -6,6 +6,7 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.indexExchange',[
 	'use strict';
 
 	var bidderName = 'indexExchange',
```

Comment:
Is it still needed?

Reply:
Yes, I would stick to this full name. Also all lines in DFP are targeted to `indexExchange`.